__test__/github-api-helper.test.ts

```diff
@@ -0,0 +1,37 @@
+import * as github from '@actions/github'
+import * as githubApiHelper from '../lib/github-api-helper'
+
+jest.mock('@actions/github')
+
+describe('github-api-helper tests', () => {
+  describe('github enterprise compatibility', () => {
+    beforeEach(() => {
+      process.env.GITHUB_SERVER_URL = 'https://enterprise.git.com'
+    })
+
+    afterEach(() => {
+      delete process.env.GITHUB_SERVER_URL
+    })
+
+    it('getDefaultBranch should use GITHUB_SERVER_URL to set the baseUrl', async () => {
+      ;(github.getOctokit as jest.Mock).mockImplementation(() => {
```

Comment:
Is the `;` necessary for this typecast?

Reply:
Technically no, but the prettier configuration in this repository seems to require it. At least the `format-check` fails without it and `format` adds it.

Reply:
Ah right, my mistake, it is correct. The `;` is to prevent the parentheses of `(github.getOctokit as jest.Mock)` to be interpreted as function invocation in case later statement is added above.

```
console.log('foo')
(github.getOctokit as jest.Mock)
```
would be read as `console.log('foo')(github.getOctokit as jest.Mock)`, the `;` prevents it.